tiling.js

```diff
@@ -205,6 +207,59 @@ export function disable () {
     backgroundSettings = null;
     interfaceSettings = null;
     workspaceSettings = null;
+
```

Comment:
@Lythenas, that this type of patching needs to be done in `patches.js` and using the patterns there, otherwise this wouldn't pass EGO review) - see examples there for patching prototypes (which will then be automatically reverted on PaperWM disable).

But in principal I can see what you've done here.